### PR TITLE
Quartz bricks

### PR DESCRIPTION
**Pull request by IncredibleHolg**
solves missing quartz bricks in #1807

### CODE REVIEW

File: overviewer_core/src/block_class.c [hunk 1]
```diff
@@ -214,8 +240,11 @@ const mc_block_t block_class_alt_height[] = {
     block_birch_stairs,
     block_jungle_stairs,
     block_quartz_stairs,
+    block_quartz_stairs,
```
Comment (Santobert) [inline overviewer_core/src/block_class.c:243]: Is it intentional that the line occurs twice?

**Review by Santobert — COMMENTED** [review]
To be honest, I don't really know the internals of the Minecraft Overviewer. But I would be very happy to see your changes in the main branch. Therefore, thank you very much for your efforts. Looking through your code I noticed a few things. I'm not sure whether these are actually bugs or not. But if this helps to speed up the review process, I'm happy to help.
Please ignore my comments, if they are not useful :smirk:

**Comment by IncredibleHolg** [comment]
Hi,
I removed the duplicate lines. And updated the last commit using --amend. 
The moste recent sha1 is 6b2be16

**Comment by dreamwraith** [comment]
hi @IncredibleHolg  - I believe I was able to merge your various changes and get them all into a single branch here:

https://github.com/dreamwraith/Minecraft-Overviewer/tree/1.16_changes

**Comment by CounterPillow** [comment]
Sorry for the long wait, I finally got around to merging this.

Not sure why it took me this long, the code looked good and tested okay. Let me know if there are any further problems.

**Pull request merged**